README.md

```diff
@@ -691,6 +697,19 @@ Proceedings of the ACM on Interactive, Mobile, Wearable and Ubiquitous Technolog
 *[EyeGraph: Modularity-aware Spatio Temporal Graph Clustering for Continuous Event-based Eye Tracking](https://openreview.net/forum?id=YxuuzyplFZ)*,  
 Advances in Neural Information Processing Systems (NeurIPS) Datasets and Benchmarks Track, 2024. [Project Page](https://eye-tracking-for-physiological-sensing.github.io/eyegraph/), [PDF](https://openreview.net/pdf?id=YxuuzyplFZ), [Supplementary Materials](https://openreview.net/attachment?id=YxuuzyplFZ&name=supplementary_material).
 
+
```

Comment:
the additional spacing here and in the previous list is messing up with the changes. Please always see the preview of the changes.
<img width="928" height="386" alt="image" src="https://github.com/user-attachments/assets/434c8ea8-6173-45f2-b14c-e29eb2a8ea84" />

And use line breaks, so that there is 1 line for the authors, 1 line for the title, and 1 line for the publication venue. See the preview of other entries.